.github/workflows/scripts/pre-release/references.sh

```diff
@@ -32,3 +33,19 @@ if [[ "$results" != "" ]]; then
     exit 1
 fi
 
+# Verify documentation refers to the most recent release tag
+# TODO(https://github.com/slsa-framework/slsa-github-generator/issues/409):
+#   Include "./internal/builders/container/*" in this check
+results=$(
+    find . -name "*.md" -not -path ./internal/builders/container/* -print0 \
```

Comment:
We're basically about the GA the container builder so you can just remove the exception for that builder.
```suggestion
    find . -name "*.md" -print0 \
```